core/src/main/java/org/mskcc/cbio/portal/scripts/ImportTabDelimData.java

```diff
@@ -148,7 +149,7 @@ public void importData(int numLines) throws IOException, DaoException {
                 }
             } else if (treatmentProfile) {
                 if (treatmentIdIndex == -1) {
-                    throw new RuntimeException("Error: the following column should be present for treatment response data: treatment_id");
+                    throw new RuntimeException("Error: the following column should be present for treatment response data: stable_id");
```

Comment:
maybe replace `present for treatment response data` with `present for this type of data`?

Reply:
Message was updated.